typescript/infra/config/environments/testnet3/middleware/liquidity-layer/addresses.json

```diff
@@ -2,23 +2,28 @@
   "goerli": {
     "circleBridgeAdapter": "0xfe9d88aA85c5917822C804b949BcEDE832C02ce2",
     "portalAdapter": "0x68D753982e89CC083917863F6dc9738448B91ef9",
-    "router": "0x2abe0860D81FB4242C748132bD69D125D88eaE26"
+    "proxyAdmin": "0x8f919348F9C4619A196Acb5e377f49E5E2C0B569",
```

Comment:
I share your intuition that this is an odd place for this but not strongly opposed to punting on moving it to SDK

Reply:
If we're shipping the LL, why not put those addresses in the SDK?

Reply:
The way the artifacts work would imo require to put in the adapter contract addresses as well at the top level which feels out of place in the SDK artifacts. Definitely not opposed to moving it in, it just felt out of place there for me too.

Reply:
IIRC, not without changing the infra/scripts/deploy.ts to account for this edge case which did not seem preferable